core/chains/evm/client/rpc_client.go

```diff
@@ -695,6 +712,28 @@ func (r *rpcClient) LatestFinalizedBlock(ctx context.Context) (head *evmtypes.He
 	return
 }
 
+func (r *rpcClient) LatestBlock(ctx context.Context) (head *evmtypes.Head, err error) {
```

Comment:
This duplicates `r.BlockByNumber(ctx, nil)`

Reply:
nit: as it's not used externally, might be a good idea to make this method private